cli/command/trust/common_test.go

```diff
@@ -0,0 +1,33 @@
+package trust
+
+import (
+	"testing"
+
+	"github.com/docker/cli/cli/trust"
+	"github.com/theupdateframework/notary/client"
+	"github.com/theupdateframework/notary/tuf/data"
+	"gotest.tools/assert"
+	is "gotest.tools/assert/cmp"
+)
+
+func TestMatchReleasedSignaturesSortOrder(t *testing.T) {
+	var releasesRole = data.DelegationRole{BaseRole: data.BaseRole{Name: trust.ReleasesRole}}
```

Comment:
nit: releasesRole := data.DelegationRole{BaseRole: data.BaseRole{Name: trust.ReleasesRole}}

Reply:
Thank you for the review. Seems the PR was merged before I modify this.

Reply:
No worries; I looked at the comment, and thought it to be not relevant enough to update before merging 🤗